src/python/test/test_dxclient.py

```diff
@@ -3710,7 +3696,7 @@ def test_dx_run_clone_analysis(self):
         finally:
             run("dx rmproject -y " + other_proj_id)
 
-    @unittest.skipUnless(testutil.TEST_RUN_JOBS, 'skipping test that runs jobs')
+    @unittest.skip("Skipping per DEVEX-2195")
```

Comment:
@commandlinegirl shall we create another ticket to correct these tests later? (since these tests also have other functionality tested)

Other than that looks good to me. Thanks!

Reply:
Yes, doing it now.